bin/build-drivers/src/build_drivers/lint_manifest_file.clj

```diff
@@ -19,6 +19,9 @@
 (s/def ::description string?)
 (s/def ::info (spell/keys :req-un [::name ::version ::description]))
 
+(s/def ::address string?)
+(s/def ::contact-info (spell/keys :req-un [::name ::address]))
```

Comment:
I'm not sure if it is, but if the contact info address is optional, this is how I'd write it:

```suggestion
(s/def ::address string?)
(s/def ::contact-info (spell/keys :req-un [::name]
                                  :opt-un [::address]))
```